.cspell.json

```diff
@@ -0,0 +1,2707 @@
+{
+    "version": "0.2",
+    "language": "en-us",
+    "words": [],
+    "ignoreWords": [
```

Comment:
We should probably put all our `.spelling` into a [custom dictionary](https://cspell.org/docs/dictionaries/custom-dictionaries).

Bonus marks if it can remain in the current location/format